pkg/turndown/turndownendpoints.go

```diff
@@ -62,26 +49,29 @@ func (te *TurndownEndpoints) HandleStartSchedule(w http.ResponseWriter, r *http.
 
 	if r.Method == http.MethodGet {
 		schedule := te.scheduler.GetSchedule()
-		if schedule == nil {
-			w.Write(wrapData(struct{}{}, nil))
+		log.Infof("get request")
```

Comment:
I'm guessing this line was accidentally committed?

Reply:
Yes it was :) thanks